### ISSUE TITLE
Arduino not renaming

### ISSUE THREAD

**Issue by edn9** [justbuchanan/i3scripts#10]
> pacman -Ss arduino
> community/arduino 1:1.8.5-1 [instalado]
Manjaro i3.

Looks like Arduino its not working, I used this command:

> ~ $ xprop | grep WM_CLASS
> WM_CLASS(STRING) = "sun-awt-X11-XFramePeer", "processing-app-Base"

So I put it in the autorename_script, but I still get the default icon.

>     'processing-app-Base':'',
>     'sun-awt-X11-XFramePeer':'',
>     'arduino':'',
>     'JAVA_TOOL_OPTIONS':'',
> 
> DEFAULT_ICON = '*'

The Icon I made and the font its already working in other programs.
Font: https://github.com/necronoise/necrofont

**Comment by justbuchanan** [comment]
I took a look and the issue here is that the script does a case-insensitive lookup to find the correct icon. It does this by requiring that the window class names in `WINDOW_ICONS` are lowercase, then does something like `icon = WINDOW_ICONS[WM_CLASS.lowercase()]`.

If you change 'sun-awt-X11-XFramePeer' to 'sun-awt-x11-xframepeer', it should work.

I added a check in https://github.com/justbuchanan/i3scripts/commit/14014e8ad182752a20966629c15e81c3eed60664 that should make this clearer - it will now print out an error and exit the program if you attempt to add a window class that isn't lowercase.
